Vagrantfile

```diff
@@ -7,18 +7,25 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.ssh.username = "vagrant"
   config.ssh.forward_agent = true
   config.vm.provision :shell, :path => "vagrant_setup.sh", :privileged => false
-  # Sync a folder between the host and all guests.
-  # Uncomment this line (and adjust as you like)
-  #config.vm.synced_folder "~/code", "/code"
 
   config.vm.define "pwn", primary: true do |u64|
-    u64.vm.box = "geerlingguy/ubuntu1604"
-    u64.vm.network "private_network", ip: "10.10.10.10"
-    u64.vm.provider "virtualbox" do |vb|
-      vb.name = "pwn ubuntu"
+    u64.vm.provider "virtualbox" do |vb, override|
+      override.vm.box ="geerlingguy/ubuntu1604"
+      override.vm.network "private_network", ip: "10.10.10.10"
+      # Sync a folder between the host and all guests.
```

Comment:
I am sure both providers would want a shared directory. Keep it commented out but let's make it not unique to the provider, unless libvirt can't do that?